### PR TITLE
[Offline] online route

### PR DESCRIPTION
**Pull request by Donhusum**
## Description

Just added a route used to check if the server can be reached, nothing special. 

## Changes

New route _utilityRoutes_ 

## Related Issues

List any related issues or tickets that this pull request addresses or resolves.

## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Any potential security implications have been considered and addressed.
- [x] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)

Non

## If mobile/frontend pull request, what version of the backend is it stable, and running on? 

Not mobile/frontend

## Notes for Reviewers

Non

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +38,7 @@
     "ejs": "^3.1.5",
     "eslint": "^8.50.0",
     "express": "^4.17.1",
+    "form-data": "^4.0.0",
```
Comment (ITPini) [inline package.json:41]: Why is form-data introduced in this PR?

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "ajv": "^8.11.0",
     "ajv-formats": "^2.1.1",
     "aws-sdk": "^2.1083.0",
+    "axios": "^1.6.0",
```
Comment (ITPini) [inline package.json:31]: Why is axios introduced in this PR?

**Review by JackSMunn — APPROVED** [review]
gj

File: __tests__/routes/utilityRoutes.test.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+const server = app.listen(PORT)
+
+
+describe('Test online response', () => {
+
+    it('Test if online', async () => {
+        const response = await request(`http://localhost:${PORT}`)
+            .get('/api/utility/online')
+            .expect(200);
+        console.log(response.body)
```
Comment (Mafusn) [inline __tests__/routes/utilityRoutes.test.js:30]: Remove this console.log

**Review by Mafusn — APPROVED** [review]
Looks good, only one console.log that has to be removed

**Review by ITPini — APPROVED** [review]

**Pull request merged**